daemon/daemon_unix.go

```diff
@@ -159,6 +159,21 @@ func getBlkioWeightDevices(config containertypes.Resources) ([]specs.WeightDevic
 	return blkioWeightDevices, nil
 }
 
+func parseAndSetSecurityOpt(daemon *Daemon, container *container.Container, config *containertypes.HostConfig) error {
```

Comment:
My main point here is that passing daemon in as an argument defeats the purpose of having this as a stand-alone function.
This might as well be
```go
func(d *Daemon) parseSecurityOpt(...) {
    // do stuff
    parseSecurityOpt(...)
}